spotinst/oceancd_verification_template_metrics/fields_spotinst_oceancd_verification_template_metrics.go

```diff
@@ -0,0 +1,1633 @@
+																							string(ContinerName): {
+																								Type:     schema.TypeString,
+																								Required: true,
+																							},
+																							string(Command): {
+																								Type:     schema.TypeList,
+																								Required: true,
+																								Elem:     &schema.Schema{Type: schema.TypeString},
+																							},
+																						},
+																					},
+																				},
+																			},
+																		},
+																	},
+																},
+															},
+														},
+													},
+												},
+											},
+										},
+									},
+								},
+								string(Jenkins): {
+									Type:     schema.TypeList,
+									Optional: true,
+									MaxItems: 1,
+									Elem: &schema.Resource{
+										Schema: map[string]*schema.Schema{
+											string(JenkinsInterval): {
+												Type:     schema.TypeString,
+												Required: true,
+											},
+											string(PipelineName): {
+												Type:     schema.TypeString,
+												Required: true,
+											},
+											string(Timeout): {
+												Type:     schema.TypeString,
+												Required: true,
+											},
+											string(TlsVerification): {
+												Type:     schema.TypeBool,
+												Optional: true,
+											},
+											string(JenkinsParameters): {
+												Type:     schema.TypeSet,
+												Optional: true,
+												MaxItems: 1,
+												Elem: &schema.Resource{
+													Schema: map[string]*schema.Schema{
+														string(ParameterKey): {
+															Type:     schema.TypeString,
+															Required: true,
+														},
+														string(ParameterValue): {
+															Type:     schema.TypeString,
+															Required: true,
+														},
+													},
+												},
+											},
+										},
+									},
+								},
+								string(Web): {
+									Type:     schema.TypeList,
+									Optional: true,
+									MaxItems: 1,
+									Elem: &schema.Resource{
+										Schema: map[string]*schema.Schema{
+											string(Body): {
+												Type:     schema.TypeString,
+												Optional: true,
+											},
+											string(Insecure): {
+												Type:     schema.TypeBool,
+												Optional: true,
+											},
+											string(JsonPath): {
+												Type:     schema.TypeString,
+												Optional: true,
+											},
+											string(Method): {
+												Type:     schema.TypeString,
+												Optional: true,
+											},
+											string(Url): {
+												Type:     schema.TypeString,
+												Required: true,
+											},
+											string(TimeoutSeconds): {
+												Type:     schema.TypeInt,
+												Optional: true,
+												Default:  10,
+											},
+											string(WebHeader): {
+												Type:     schema.TypeSet,
+												Optional: true,
+												MaxItems: 1,
```

Comment:
MaxItems not required as this is array of objects

Reply:
Removed